enumap.py

```diff
@@ -148,7 +149,59 @@ def _raise_invalid_args(cls, values, mapping, names):
                 f"missing keys {missing}; invalid keys {invalid}")
 
 
-_FILL = object()  # sentinal for missing values in sparse collections
+class default(enum.auto):
+    """A subclass of enum.auto that
+
+    1. behaves as a unqiue enum member because enum members that aren't unique
+       effectively become aliases
+    2. gives the user a way of signaling that an enum value should be used as
+       a default in the collections created by SparseEnumap.map() or .tuple()
+
+    Sample usage:
+
+        >>> class Pets(SparseEnumap):
+        ...    dogs: int = default(3)
+        ...    cats: int = default(44)
+        ...    squirrels: float = 3  # this isn't a default at all
+
+        >>> Pets.tuple()
+        Pets_tuple(dogs=3, cats=44, squirrels=None)
+    """
+    _value = (enum.auto.value, None)
```

Comment:
Why does this line exist?

Reply:
I was copying the `auto` implementation, but the `enum` module is kind of a mess.